projects/clamav/build.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash -eu
+# Copyright (C) 2018 Cisco Systems, Inc. and/or its affiliates. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+export CXXFLAGS="-std=c++11 -stdlib=libc++ -O2 -fno-omit-frame-pointer -g -fsanitize=address -fsanitize-coverage=edge,trace-pc-guard,indirect-calls,trace-cmp,trace-div,trace-gep"
+export CFLAGS="-O2 -fno-omit-frame-pointer -g -fsanitize=address -fsanitize-coverage=edge,trace-pc-guard,indirect-calls,trace-cmp,trace-div,trace-gep"
```

Comment:
What's thte reason for settting these flags? The OSS-Fuzz harness should be setting all the necessary flags.

Reply:
Err, just realized you asked about this in the PR. No I don't think these are necessary at all; in fact thtey probably break UBSAN and MSAN. Was there sometthing tthat made you think they were required?

Reply:
When I first did testing with your PR I used something similar to make my own standalone fuzz targets.  This is vestigial, I think, from that build script.  I'll remove it.